airflow/providers/amazon/aws/example_dags/example_s3_bucket.py

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+
+from airflow.models.dag import DAG
+from airflow.operators.python import PythonOperator
```

Comment:
@potiuk This line is failing the backport tests "saying cannot import" -- any idea why?

https://github.com/apache/airflow/pull/8895/checks?check_run_id=716706159#step:6:6573

Reply:
very interesting.

I've never seen it before but seems like bowler does not like the file:
```
Bowler exception during transform of ./airflow/providers/amazon/aws/example_dags/example_s3_bucket.py: Transforms generated invalid CST for ./airflow/providers/amazon/aws/example_dags/example_s3_bucket.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/bowler/tool.py", line 145, in processed_file
    new_tree = self.driver.parse_string(new_text)
  File "/usr/local/lib/python3.6/site-packages/fissix/pgen2/driver.py", line 102, in parse_string
    return self.parse_tokens(tokens, debug)
  File "/usr/local/lib/python3.6/site-packages/fissix/pgen2/driver.py", line 71, in parse_tokens
    if p.addtoken(type, value, (prefix, start)):
  File "/usr/local/lib/python3.6/site-packages/fissix/pgen2/parse.py", line 164, in addtoken
    raise ParseError("bad input", type, value, context)
fissix.pgen2.parse.ParseError: bad input: type=12, value=',', context=('', (56, 36))

The above exception was the direct cause of the following exception:
```

@turbaszek -> have you seen anything like that before?

Reply:
I could  not find anything strange in the file. My hunch is that it was a transient error. Bowler uses multi-processing heavily so this might be result of lack of memory which might lead us to add some limitations to it. I am testing it locally to see if that could be a transient issue.

Reply:
Nope. Same for me :) ... That's an interesting one

Reply:
OK. I know. We are adding python_context as extra parameter of PythonOperator call and we end up with double coma:

Reply:
```
     add_keys_to_bucket = PythonOperator(
         task_id="s3_bucket_dag_add_keys_to_bucket",
-        python_callable=upload_keys
+        python_callable=upload_keys,,
+        provide_context=True
     )
```


For now you can remove coma but I will fix it in our refactor code. @randr97

Reply:
Permanent fix on the way @randr97 https://github.com/apache/airflow/pull/9064